app/scripts/modules/core/delivery/manualExecution/manualPipelineExecution.controller.js

```diff
@@ -87,11 +98,25 @@ module.exports = angular.module('spinnaker.core.delivery.manualPipelineExecution
       if (pipeline.parameterConfig && pipeline.parameterConfig.length) {
         selectedTrigger.parameters = this.parameters;
       }
-      $uibModalInstance.close(command);
+
+      if(this.pipelineIsDirty()) {
+        pipelineConfigService.savePipeline(pipeline)
+          .then(() => $uibModalInstance.close(command) );
+      } else {
+        $uibModalInstance.close(command);
+      }
+
```

Comment:
How does this happen? Is it that we're saving the rollback values in the pipeline config or something?